specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json

```diff
@@ -0,0 +1,882 @@
+      "properties": {
+        "impactedRegion": {
+          "type": "string",
+          "description": "Impacted region name."
+        },
+        "status": {
+          "type": "string",
+          "description": "Current status of event in the region.",
+          "enum": [
+            "Active",
+            "Resolved"
+          ],
+          "x-ms-enum": {
+            "name": "EventStatusValues",
+            "modelAsString": false
+          }
+        },
+        "impactedSubscriptions": {
+          "type": "array",
+          "description": "List subscription impacted by the service health event.",
+          "items": {
+            "type": "string",
+            "description": "Subscription impacted by the service health event."
+          }
+        },
+        "impactedResourcesCount": {
+          "type": "integer",
+          "description": "Total Azure resources impacted by the service health event."
+        },
+        "lastUpdateTime": {
+          "type": "string",
+          "description": "It provides the Timestamp for when the last update for the service health event.",
+          "format": "date-time"
+        }
+      }
+    },
+    "availabilityStatusListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/availabilityStatus"
+          },
+          "description": "The list of availabilityStatuses."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of availabilityStatuses. Call ListNext() with this URI to fetch the next page of availabilityStatuses."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The List availabilityStatus operation response."
+    },
+    "availabilityStatus": {
+      "type": "object",
+      "description": "availabilityStatus of a resource.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Azure Resource Manager Identity for the availabilityStatuses resource."
+        },
+        "name": {
+          "type": "string",
+          "description": "current."
+        },
+        "type": {
+          "type": "string",
+          "description": "Microsoft.ResourceHealth/AvailabilityStatuses."
+        },
+        "location": {
+          "type": "string",
+          "description": "Azure Resource Manager geo location of the resource."
+        },
+        "properties": {
+          "type": "object",
+          "description": "Properties of availability state.",
+          "properties": {
+            "availabilityState": {
+              "type": "string",
+              "description": "Availability status of the resource. When it is null, this availabilityStatus object represents an availability impacting event",
+              "enum": [
+                "Available",
+                "Unavailable",
+                "Degraded",
```

Comment:
Added enum value. Breaking change.

Reply:
What is action item on us here?

Reply:
Not necessarily anything, but:
- any SDKs/CLIs derived from this Swagger will have to bump their major version
- you may wanna consider changing `modelAsString` to `true` since that would make enum-additions no breaking change in the future

Reply:
Got it, Thanks